contracts/utils/MultiRateLimited.sol

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "../refs/CoreRef.sol";
+import "./RateLimited.sol";
+import "./IMultiRateLimited.sol";
+import "@openzeppelin/contracts/utils/math/Math.sol";
+
+/// @title contract for putting a rate limit on how fast an address can perform an action e.g. Minting
+/// there are two buffers, one buffer which is each individual addresses's current buffer,
+/// and then there is a global buffer which is the buffer that each individual address must respect as well
+/// @author Fei Protocol
+contract MultiRateLimited is RateLimited, IMultiRateLimited {
+
+    /// @notice the rate per second for each address
+    mapping (address => uint256) public individualRateLimitPerSecond;
```

Comment:
Should these `public` state variables not have the `override` specifier applied? 

They'll generate getters and you're overriding `function individualRateLimitPerSecond(address) external view returns(uint256);` etc on the inherited interface